code/modules/food_and_drinks/recipes/food_mixtures.dm

```diff
@@ -157,7 +157,7 @@
 
 /datum/chemical_reaction/food/pancakebatter
 	results = list(/datum/reagent/consumable/pancakebatter = 15)
-	required_reagents = list(/datum/reagent/consumable/eggyolk = 12, /datum/reagent/consumable/milk = 10, /datum/reagent/consumable/flour = 5)
+	required_reagents = list(/datum/reagent/consumable/eggyolk = 15, /datum/reagent/consumable/milk = 10, /datum/reagent/consumable/flour = 5)
```

Comment:
I mean it's minor as fuck, but why?

Reply:
It was weird having it be the only eggception to the 5u egg thing. Everything is kind of uniform now.

Reply:
aight let's jive